antora-ui-camel/src/css/header.css

```diff
@@ -318,6 +320,12 @@ body {
     padding: 1rem 2rem 1rem 0;
   }
 
+  @media screen and (max-width: 1080px) {
+    .navbar-item {
+      padding-right: 1.5rem;
+    }
+  }
```

Comment:
I'd change this in higher widths, no need for a lot of padding, 1.5rem should look okay in all resolutions.